src/components/CreateQuizButton/index.tsx

```diff
@@ -3,15 +3,30 @@ import { Button } from '@mantine/core';
 import Icon from '../../components/Icon';
 import { useNavigate } from 'react-router';
 
-function CreateQuizButton({ userName }): JSX.Element {
+type Props = {
+  userName: string;
+  isFloating?: boolean;
+};
+
+function CreateQuizButton({ userName, isFloating = false }: Props): JSX.Element {
   const navigate = useNavigate();
 
   return (
     <Button
-      size="md"
+      size={isFloating ? 'xl' : 'lg'}
       onClick={() => navigate(`/configure-quiz/${userName}`)}
       variant="filled"
-      sx={(theme) => ({ boxShadow: theme.shadows.sm })}
+      sx={(theme) => ({
+        boxShadow: theme.shadows.sm,
+        ...(isFloating
```

Comment:
can you use `Affix` for floating button
https://mantine.dev/core/affix/

Reply:
Sure. I can try that.